ben_lowenstein.cfdg

```diff
@@ -0,0 +1,30 @@
+startshape OUTER
+
+rule OUTER {
+    COLORC { b 1 sat 1 }
+}
+
+rule COLORC 5 {
+    CIRCLE { size 1 hue 10 }
+    COLORC { size .99 hue 10 r 38 }
+}
+
+rule COLORC 100 {
+    CIRCLE { x .7 y .7 size .2 hue 10 }
+    COLORC { size .992 hue 10 r -70 }
+}
+
+rule COLORC {
+    MAYBEC { size .6 x -.7 y -.7 }
+    MAYBEC { size .6 x -.7 y .7 }
+    MAYBEC { size .6 x .7 y -.7 }
+    MAYBEC { size .6 x .7 y .7 }
+}
+
+rule MAYBEC 10 {
+    COLORC { hue 60 }
+}
+
+rule MAYBEC {
+
+}
```

Comment:
I can't comment directly on your picture so I'll do that here! I really love it. The color explodes off the page and your eye is drawn to places where there is more to see. The different layers of the complex parts all go together and the simple background (both the white and the center circles) make the more complicated parts really attractive.